src/structures/GuildChannel.js

```diff
@@ -539,6 +539,17 @@ class GuildChannel extends Channel {
     return this.permissionsFor(this.client.user).has(Permissions.FLAGS.MANAGE_CHANNELS);
   }
 
+  /**
+ * Whether the channel is manageable by the client user
```

Comment:
Indentation is off (missing 3 spaces) on this line and the next 3 lines

Reply:
Thanks I missed those. Commit fixing this incoming.